src/util/sandbox/test_sandbox.c

```diff
@@ -208,12 +208,12 @@ mountns_null( void ) {
    seccomp is effective. */
 void
 seccomp_default_filter( void ) {
-  int fds[ 4 ] = { 0, 1, 2, 3 };
+  int fds[ 5 ] = { 0, 1, 2, 3, 4 };
 
   struct sock_filter seccomp_filter[ 128UL ];
   populate_sock_filter_policy_test_sandbox( 128UL, seccomp_filter );
 
-  TEST_FORK_OK( fd_sandbox( 1, getuid(), getgid(), SIZEOFA( fds ), fds, sock_filter_policy_test_sandbox_instr_cnt, seccomp_filter ); );
+  TEST_FORK_OK( fd_sandbox( 1, getuid(), getgid(), 0, SIZEOFA( fds ), fds, sock_filter_policy_test_sandbox_instr_cnt, seccomp_filter ); );
```

Comment:
It's an extremely small detail, but I think it does help to make these `0UL` so it's clear it's a count of something (which then reminds me it's the rlimit_file_cnt) and not a bool